Local

```diff
@@ -1,3 +0,0 @@
-redis: redis-server
```

Comment:
Hi there. Can you keep this Local file in case someone wants to use flask base without docker?